dev-libs/xapian/xapian-1.2.25.recipe

```diff
@@ -48,21 +47,29 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libuuid$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libuuid$secondaryArchSuffix >= 1.3
 	lib:libz$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
 	xapian${secondaryArchSuffix}_devel = $portVersion
 	devel:libxapian$secondaryArchSuffix = $libVersionCompat
+	cmd:xapian_config$secondaryArchSuffix = $portVersion
 	"
 REQUIRES_devel="
 	xapian$secondaryArchSuffix == $portVersion base
 	"
+if [ -n "$secondaryArchSuffix" -a "$commandBinDir" != "$prefix"/bin ]; then
+CONFLICTS_devel="
+	xapian_devel
```

Comment:
Sorry, but I don't understand this. Why would `xapian${secondaryArchSuffix}_devel` conflicts with `xapian_devel`?

Reply:
This conflicts is needed because we have moved `$libDir/cmake/xapian/*` to `$dataDir/cmake/Modules/xapian/*`.
This is not an issue on x86 secondary arch **_if_** the primary arch is x86_gcc2.

But since x86 secondary arch will become available for a x86_64 hybrid, we need to make sure that, on that hybrid arch, `xapian_x86_devel` and `xapian_devel` will never be activated together.
That's because these packages have overlapping `$dataDir/cmake/Modules/xapian/*` files which have conflicting contents.